### PR TITLE
[Feature] Print lake table's metadata in json by meta tool

### PR DESCRIPTION
**Pull request by sduzh**
## What type of PR is this：
- [ ] bugfix
- [x] feature
- [ ] enhancement
- [ ] refactor
- [ ] others

## Example usage
``` shell
>  ossutil64 cat oss://bucket/65a57777-b63f-478d-a7d2-7f209525825c/16017/meta/0001000000001394_0000000000000007.meta | head -n 1 | tr -d '\n' |  output/be/bin/meta_tool.sh --operation=print_lake_metadata
{
    "id": 281474976715668,
    "version": 7,
    "schema": {
        "keys_type": "DUP_KEYS",
        "column": [
            {
                "unique_id": 0,
                "name": "c0",
                "type": "INT",
                "is_key": true,
                "aggregation": "NONE",
                "is_nullable": true,
                "precision": 0,
                "frac": 0,
                "length": 4,
                "index_length": 4
            }
        ],
        "num_short_key_columns": 1,
        "num_rows_per_row_block": 1024,
        "compress_kind": "COMPRESS_LZ4",
        "next_column_unique_id": 1,
        "compression_type": "LZ4_FRAME",
        "id": 16018
    },
    "rowsets": [
        {
            "id": 2,
            "overlapped": false,
            "segments": [
                "f6e423e9-d97b-4a19-b5a0-e541eb7b4c34.dat"
            ],
            "num_rows": 1,
            "data_size": 232
        },
        {
            "id": 3,
            "overlapped": false,
            "segments": [
                "7b095819-4481-4f8e-aea5-0f177dbfee52.dat"
            ],
            "num_rows": 1,
            "data_size": 233
        }
    ],
    "next_rowset_id": 4,
    "cumulative_point": 1
}
```

### CODE REVIEW

File: be/src/tools/meta_tool.cpp [hunk 1]
```diff
@@ -72,7 +73,7 @@ const std::string HEADER_PREFIX = "tabletmeta_";
 DEFINE_string(root_path, "", "storage root path");
 DEFINE_string(operation, "get_meta",
               "valid operation: get_meta, flag, load_meta, delete_meta, delete_rowset_meta, "
-              "show_meta, check_table_meta_consistency");
+              "show_meta, check_table_meta_consistency print_lake_metadata print_lake_txn_log");
```
Comment (tracymacding) [inline be/src/tools/meta_tool.cpp:76]: miss "," ?

File: be/src/tools/meta_tool.cpp [hunk 2]
```diff
@@ -559,6 +562,36 @@ int meta_tool_main(int argc, char** argv) {
             return -1;
         }
         show_segment_footer(FLAGS_file);
+    } else if (FLAGS_operation == "print_lake_metadata") {
+        starrocks::lake::TabletMetadataPB metadata;
+        if (!metadata.ParseFromIstream(&std::cin)) {
+            std::cerr << "Fail to parse tablet metadata";
```
Comment (tracymacding) [inline be/src/tools/meta_tool.cpp:568]: std::endl?

**Review by tracymacding — DISMISSED** [review]

File: be/src/tools/meta_tool.cpp [hunk 3]
```diff
@@ -82,6 +83,7 @@ DEFINE_string(json_meta_path, "", "absolute json meta file path");
 DEFINE_string(pb_meta_path, "", "pb meta file path");
 DEFINE_string(tablet_file, "", "file to save a set of tablets");
 DEFINE_string(file, "", "segment file path");
+DEFINE_string(file, "", "segment file path");
```
Comment (wyb) [inline be/src/tools/meta_tool.cpp:86]: duplicated.

**Review by wyb — APPROVED** [review]

**Comment by sduzh** [comment]
run starrocks_admit_test

**Pull request merged**